include/netcdf.h

```diff
@@ -161,6 +161,7 @@ Use this in mode flags for both nc_create() and nc_open(). */
 
 #define NC_PERSIST       0x4000  /**< Save diskless contents to disk. Mode flag for nc_open() or nc_create() */
 #define NC_INMEMORY      0x8000  /**< Read from memory. Mode flag for nc_open() or nc_create() */
+#define NC_HDF5_SWMR     0x2000 /** Reuse deprecated MPIIO flag for SWMR **/
```

Comment:
Rather than reusing `NC_MPIIO` flag, maybe this should be `0xA000`. I don't think this would clash with anything?